### ISSUE TITLE
Agymah's Lair Issues: Flagging For RVR, Stage 2 not showing up correctly, MOBs not respawning

### ISSUE THREAD

**Issue by Delegator2** [WarEmu/WarBugs#18383]
I'm trying to solo the Agymah's Lair PQ (Easy level 18-19 Order in Marshes of Madness). After finally finding the entrance -- no small feat! -- I have run into several strange issues. I don't know whether these are intertwined, but they are all about this PQ so entering as one issue.

1. When I entered the PQ, it flagged me for RVR despite this PQ being nowhere near any RVR zone. I confirmed this happened to others as well.
2. I killed 30 tomb warriors, and the quest is sat there at 30/30 in stage 1, not moving forward to stage 2
3. I ended up destroying casket shards and killing the tomb guards that spawned in order to try to move it along. I had to kill the spawn of guards multiple times, and then that got me to stage three. It turned out that the casket shards were for stage 3, so I had to go back and destroy them (and the priests that now spawned) again. 
4. That let me finish the quest. I exited.
5. Killing MOBs outside the PQ (without the PQ showing on screen) gave me influence. This seems a UI issue -- when I then entered the PQ area, the count of MOBs killed had clearly been incrementing.
6. When I restarted the quest, I killed all the stage 1 MOBs and there were only 19/30. And they haven't respawned after more than 15 minutes, so I can't keep going

In short, this PQ seems rife with problems.

![Influence from MOB outside the PQ (PQ doesn't show in UI)](https://user-images.githubusercontent.com/77587188/105225257-4ccb1b80-5b2c-11eb-8dd2-a6b9aba8d5c6.jpg)
![Getting flagged for RVR for entering Agymah's Lair](https://user-images.githubusercontent.com/77587188/105225565-c400af80-5b2c-11eb-8868-a48c6f6c3b21.png)
![Flagged for RVR now](https://user-images.githubusercontent.com/77587188/105225568-c5ca7300-5b2c-11eb-9b2d-c847923f478c.png)

The strange thing is that Stage 2 never showed in the UI. So, I don't know exactly how many tomb guards I had to kill, nor do I know whether Stage 2 was timed. 

After completing the PQ I did it again...same thing including getting flagged for RVR.
